docs/protocol/wallets/subwallet.md

```diff
@@ -64,6 +64,62 @@ To learn more about SubWallet visit their [Website](https://subwallet.app).
 
 8. Congratulations you have fully created your substrate wallet with subwallet! 
 
+## Adding New Networks to SubWallet
+
+:::caution
+
+This tutorial assumes you have already completed the setup for SubWallet. If you have not please refer to the **[section above](#create-or-import-a-new-wallet)**
+
+:::
+
+SubWallet supports adding custom networks. This can be helpful for in-development networks such as the Subspace Network as we have regularly changing RPC points and versions of testnets, and soon various domains. Below you will find a simple guide on how to add these new networks. Additionally we will try and keep an updated list of active RPC points below for development. 
+
+:::info RPC Endpoints
+
+- **Gemini 3d Endpoint: ** wss://rpc.gemini-3d.subspace.network/ws
+- **Gemini 3e Endpoint: ** wss://rpc.gemini-3e.subspace.network/ws
+
+
+:::
+
+1. Open SubWallet, Select the 3 Line menu in the top left of the wallet
+
+    ![rpc-step-1](/img/doc-imgs/subwallet/rpc-step-1.png)
+
+2. Inside the settings menu you will see the option to *Manage Networks*, click this.
+
+    ![rpc-step-2](/img/doc-imgs/subwallet/rpc-step-2.png)
+
+3. Within the *Manage Netowrks* menu you will see a `+` in the top right corner, click this
+
+    ![rpc-step-3](/img/doc-imgs/subwallet/rpc-step-3.png)
+
+4. This will open the *Import Network* menu, where you will see a few options
+    - Provider URL
+    - Network Name
+    - Symbol
+    - Blockexplorer
+    - Crowdloan URL
+
+    The only option that is required is the Provider URL. You can add an explorer if you would like but it is not required. (The current Subspace Explorer is available [here](https://explorer.subspace.network).
```

Comment:
Missing close parenthesis. Or unnecessary opening one. Does the integration work with our block explorer?

Reply:
Resolved with commit 5ca0e73